src/pages/pages/what_we_do.hbs

```diff
@@ -0,0 +1,97 @@
+---
+title: What We Do
+layout: toolkit.blank
+---
+
+{{{embed "patterns.components.sky.base"}}}
+
+<main role="main" id="main">
+  {{#embed "patterns.components.sky.base" class="Sky--clouds"}}
+    {{#content "top"}}
+      <div class="Grid Grid--alignMiddle u-spaceBottom1">
+        <div class="Grid-cell u-md-size1of2 u-padSides1 u-md-padSidesNone u-md-flexOrderLast u-textCenter">
+          <img class="u-limitHeight2of3" src="/images/sandbox/what_we_do-robot.svg" alt="" style="width: 21.25em; height: 15.625em;">
+        </div>
+        <div class="Grid-cell u-md-size1of2 u-padSides1 u-padBottom1 u-md-padTop1">
+          <h1 class="u-textGrow5 u-sm-textGrow6 u-spaceRight1">
+            What We Do
+          </h1>
+          <p class="u-textGrow1">
+            We love solving tough puzzles through our design <span class="u-textNoWrap">and code</span>
+          </p>
+        </div>
+      </div>
+    {{/content}}
+  {{/embed}}
+  <div class="u-bgWhite">
+    <div class="u-containProse u-pad1 u-spaceItems1">
+      <p class="u-textGrow1">
+        Do you have an app or site that no one believes can be made responsive? Perhaps you want to use the latest web technology, but no one on your team has experience with&nbsp;it? We Can Help.
+      </p>
+      <p>
+        Many companies know they need a responsive web design or progressive web app, but don't know where to start. Their teams face the pressure of deadlines but need the project done right the first time.
+      </p>
+      <p>
+        We work closely with internal teams to find the best design and technical approach. We set them up for success by incorporating education and training into every project.
+      </p>
+    </div>
+
+    <div class="Grid Grid--withGutter u-spaceItems1 u-containSpread u-pad1">
```

Comment:
`u-pad1` and `Grid--withGutter` contradict each other. Consider separating these out into separate divs, perhaps one to contain and pad and the grid for layout.